### ISSUE TITLE
Contao framework not initialized upon invalid route

### ISSUE THREAD

**Issue by leofeyer** [contao/core-bundle#71]
If you enter a URL that is not matched by a route (e.g. `http://contao.local/`), the Contao framework does not get initialized:

```
Fatal error: Class 'Contao\Frontend' not found in ~/contao/standard-edition/vendor/contao/core-bundle/src/EventListener/OutputFromCacheListener.php on line 31
```

The problem is that the Symfony exception listener is a sub request and we have recently added the following lines:

```php
// InitializeSystemListener::onKernelRequest()
if (!$event->isMasterRequest()) {
    return;
}
```

@aschempp How do we handle this case?

**Comment by leofeyer** [comment]
This is related to contao/contao#64.

**Comment by aschempp** [comment]
There can never be a route that is not matched? Everything will/should be matched by the `FrontendLoader`?

**Comment by aschempp** [comment]
I was able to reproduce this, it only happens if `prepend_locale` is `true`. However, I don't think it's because of the master request, as there is always a master request before a subrequest. As you said it's related to https://github.com/contao/contao/issues/64 so I'll take a look at that.

**Comment by aschempp** [comment]
See #76

**Comment by leofeyer** [comment]
Closed in favor of the pull request (#76).
